src/test/java/millom/sandbox/service/MessageServiceImplTest.java

```diff
@@ -0,0 +1,27 @@
+package millom.sandbox.service;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.verify;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class MessageServiceImplTest {
+  MessageServiceImpl messageServiceImpl;
+
+  @BeforeEach
+  void setUp() {
+    messageServiceImpl = new MessageServiceImpl();
+  }
+
+  @Test
+  void TestConvertCuriositySol() {
+    assertEquals(3504, messageServiceImpl.convertCuriositySol("2022-06-15"));
+  }
+
+  @Test
+  void convertCuriositySolThrowsExceptionIfDateInvalidFormat() {
+    String date = "20-06-15";
+    assertThrows(RuntimeException.class, () -> messageServiceImpl.convertCuriositySol(date));
+  }
+}
```

Comment:
A new test for `null` check could be added here if one should decide to do that. See comments in `MessageServiceImpl` class.

Reply:
Not needed for now as we agreed upon that a null check is not needed.